### DISCUSSION TITLE
Pagination / Sorting - Tables

### DISCUSSION THREAD

**Discussion by prabhushan** [CMSgov/design-system#1802] [Ideas]
Hi,

Do we have any assets for table pagination / sorting. This could be the most utilized feature in any Table / grid.

**Comment by line47** [comment]
Hey @prabhushan, we currently do not have a pagination or sorting component. If you have an example you'd like to share please feel free. thanks!

**Comment by ghost** [comment]
Hi @prabhushan , if this issue can be expounded upon please comment here and I can move this to a jira issue for tracking, otherwise I will close in 4 days. Thank you!

**Comment by rylew1** [comment]
Have a similar need to be able to click the column headers to sort, and be able to paginate. Either have to go with your own implementation or something like `react-table` 

Although op may be asking about icon assets
